ts/features/wallet/component/__test__/CardLogoPreview.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { fireEvent, render } from "@testing-library/react-native";
+import { View } from "native-base";
+import * as React from "react";
+import { CardLogoPreview } from "../CardLogoPreview";
+import anImage from "../../../../../img/wallet/cards-icons/bPay.png";
+
+describe("CardLogoPreview", () => {
+  it("should show the left component, the image and when press should run the onPress", () => {
+    const left = () => <View testID="leftComponent"></View>;
```

Comment:
```suggestion
    const left = () => <View testID="leftComponent"/>;
```

Reply:
Modified in [6faa89f](https://github.com/pagopa/io-app/pull/2878/commits/6faa89f07670a693b243ff6c5da2bac028d52962)